doc/classes/TextEdit.xml

```diff
@@ -5,64 +5,92 @@
 	</brief_description>
 	<description>
 		TextEdit is meant for editing large, multiline text. It also has facilities for editing code, such as syntax highlighting support and multiple levels of undo/redo.
+		[b]Note:[/b] Most viewport, caret and edit methods contain a [code]caret_index[/code] argument for [member caret_multiple] support. The argument should be one of the following: [code]-1[/code] for all carets, [code]0[/code] for the main caret, or [code]1-x[/code] for secondary carets
```

Comment:
`1-x` looks confusing. It would be sufficient to say that carets above 0 are secondary.